packages/runner/lib/kill.ts

```diff
@@ -1,11 +1,10 @@
 import { Result, resultOk, resultErr } from '@nangohq/shared';
-import { childProcesses } from './state.js';
+import { runningSyncsWithAborts } from './state.js';
 
 export const kill = (syncId: string): Result<string> => {
-    const child = childProcesses.get(syncId);
-    if (child) {
-        child.kill();
-        childProcesses.delete(syncId);
+    const abortController = runningSyncsWithAborts.get(syncId);
+    if (abortController) {
+        abortController.abort();
         return resultOk('cancelled');
     } else {
         return resultErr('child process not found');
```

Comment:
message is not relevant anymore.

Reply:
Thanks, updated with https://github.com/NangoHQ/nango/pull/1684/commits/56c3414df31b0d1d85396647e27f757dd9ef936e